hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/s3/multipart/TestS3MultipartRequest.java

```diff
@@ -83,6 +84,8 @@ public void setup() throws Exception {
     when(ozoneManager.getMetrics()).thenReturn(omMetrics);
     when(ozoneManager.getMetadataManager()).thenReturn(omMetadataManager);
     auditLogger = Mockito.mock(AuditLogger.class);
+    OmMetadataReader omMetadataReader = Mockito.mock(OmMetadataReader.class);
+    when(ozoneManager.getOmMetadataReader()).thenReturn(omMetadataReader);
```

Comment:
This line is duplicated with line 85. Need to verify if the mock is necessary.
`when(ozoneManager.getMetadataManager()).thenReturn(omMetadataManager);`

Reply:
They seem to be different: `getMetadataManager` vs. `getOmMetadataReader`.

Reply:
Oops, my mistake.